alerts/cluster-etcd-operator/OWNERS

```diff
@@ -0,0 +1,9 @@
+reviewers:
```

Comment:
Nit: Since cluster-etcd-operator is the place where the alerts are but not the component that the actual alerts are about I wonder if naming this directory after the component itself might be better: `alerts/etcd/<alert-name>`

Reply:
why cant this all live in etcd-operator repo?

Reply:
During last WG call we decided to name directories after whoever governs/deploys PrometheusRule object hence naming it after the operator. This way we have a clear mapping of where (which repo) alert lives. Also after OWNERS sync is established whoever has rights to change alerts also should have rights to change runbooks.

Reply:
>why cant this all live in etcd-operator repo?

Good point Sam, this will otherwise trigger another migration like with the alerts :) Might as well keep them closer to where the alerts are defined. @paulfantom @RiRa12621 what was the reason for not having them in each repo?

Reply:
It was one of the bigger conversation points during the WG call. 
The biggest win here that it's a central place for someone to browse around if they need to without having to click and find each components repository.

Additionally distributing out into separate repos is easier later on, in case we decide that's the better way, than searching them all together.

There's some more minor points, that we went over but those are the bigger ones.